### PR TITLE
1.1

### LINKED ISSUE #934: Compatibility with future 1.1 release

### PR DESCRIPTION
**Pull request by eggrobin**
Fix #934.

### CODE REVIEW

File: ksp_plugin_adapter/flight_planner.cs [hunk 1]
```diff
@@ -233,9 +235,29 @@ class FlightPlanner : WindowRenderer {
             !double.IsNaN(manoeuvre.inertial_direction.x +
                           manoeuvre.inertial_direction.y +
                           manoeuvre.inertial_direction.z)) {
-          if (guidance_node_ == null) {
+          var previous_update_mode = vessel_.orbitDriver.updateMode;
+          if (guidance_node_ == null ||
+              !vessel_.patchedConicSolver.maneuverNodes.Contains(
```
Comment (pleroy) [inline ksp_plugin_adapter/flight_planner.cs]: Is the misspelling from KSP?

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 2]
```diff
@@ -742,222 +793,164 @@ where is_in_physics_bubble(vessel)
...
+              double scale = (ScaledSpace.ScaledToLocalSpace(
+                                  MapView.MapCamera.transform.position) -
+                              position_at_ignition).magnitude * 0.015;
+              Action<XYZ, UnityEngine.Color> add_vector =
+                  (world_direction, colour) => {
+                UnityEngine.GL.Color(colour);
+                AddSegment(
+                    position_at_ignition,
+                    position_at_ignition + scale * (Vector3d)world_direction,
+                    false);
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Comment `false`.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 3]
```diff
@@ -742,222 +793,164 @@ where is_in_physics_bubble(vessel)
...
+    return PlanetariumCamera.Camera.WorldToScreenPoint(
+               ScaledSpace.LocalToScaledSpace(world));
   }
 
-  private void ResetRenderedTrajectory() {
-    DestroyRenderedTrajectory();
-    rendered_trajectory_ = NewRenderedTrajectory(XKCDColors.AcidGreen);
-    rendered_prediction_ = NewRenderedTrajectory(XKCDColors.Fuchsia);
+  private static void AddSegment(XYZSegment segment) {
+    AddSegment((Vector3d)segment.begin, (Vector3d)segment.end, true);
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Comment `true`.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 4]
```diff
@@ -742,222 +793,164 @@ where is_in_physics_bubble(vessel)
...
+      // have lengths |inner_product| / d and d - r^2/d, where d is the distance
+      // between the camera and the body and r is the body's radius, thus if
+      // |inner_product| < d^2 - r^2, |point| is above the plane passing
+      // through the horizon.
+      // Otherwise, we check whether |point| is within the cone hidden from the
+      // camera, by comparing the squared cosines multiplied by
+      // d^2|camera_to_point|^2.
+      // In addition, we check whether we're inside the body (this covers the
+      // cap above the horizon plane and below the surface of the body, which
+      // would otherwise be displayed.
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: )

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 5]
```diff
@@ -1170,9 +1160,9 @@ where is_in_physics_bubble(vessel)
   }
 
   private void KSPFeatures() {
-    display_patched_conics_ =
-        UnityEngine.GUILayout.Toggle(value : display_patched_conics_,
-                                     text  : "Display patched conics");
+     display_patched_conics_ = UnityEngine.GUILayout.Toggle(
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Your indent is bad and you should feel bad.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 3 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 4 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 5 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

**Pull request merged**